projects/packages/my-jetpack/src/products/class-protect.php

```diff
@@ -398,4 +404,40 @@ public static function get_manage_urls_by_feature() {
 	public static function is_upgradable_by_bundle() {
 		return array( 'security', 'complete' );
 	}
+
+	/**
+	 * Return site Jetpack Protect data for the REST API.
+	 *
+	 * @return WP_Rest_Response|WP_Error
+	 */
+	public static function get_site_protect_data() {
+		$scan_data = Protect_Status::get_status();
+
+		$waf_config     = array();
+		$waf_supported  = false;
+		$is_waf_enabled = false;
+
+		if ( class_exists( 'Automattic\Jetpack\Waf\Waf_Runner' ) ) {
+			// @phan-suppress-next-line PhanUndeclaredClassMethod
```

Comment:
I am suppressing these warnings because I remember a conversation about how we didn't want to add Waf_Runner as a dependency to My Jetpack as it would also add it as a dependency to all the plugins. Conversation is here (p1723736483790589-slack-C02LK1W8T4Z ), and I couldn't find a better way to get rid of these warnings